src/DeploymentManager/DeploymentManager/Az.DeploymentManager.psd1

```diff
@@ -0,0 +1,156 @@
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = 'Get-AzDeploymentManagerArtifactSource', 
+               'New-AzDeploymentManagerArtifactSource', 
+               'Set-AzDeploymentManagerArtifactSource', 
+               'Remove-AzDeploymentManagerArtifactSource', 
+               'Get-AzDeploymentManagerRollout', 
+               'Stop-AzDeploymentManagerRollout', 
+               'Restart-AzDeploymentManagerRollout', 
+               'Remove-AzDeploymentManagerRollout', 
+               'New-AzDeploymentManagerServiceTopology', 
+               'Get-AzDeploymentManagerServiceTopology', 
+               'Set-AzDeploymentManagerServiceTopology',
+               'Remove-AzDeploymentManagerServiceTopology', 
+               'Get-AzDeploymentManagerService', 
+               'New-AzDeploymentManagerService', 
+               'Set-AzDeploymentManagerService', 
+               'Remove-AzDeploymentManagerService', 
+               'Get-AzDeploymentManagerServiceUnit', 
+               'New-AzDeploymentManagerServiceUnit', 
+               'Set-AzDeploymentManagerServiceUnit', 
+               'Remove-AzDeploymentManagerServiceUnit',
+               'Get-AzDeploymentManagerStep', 
+               'New-AzDeploymentManagerStep', 
+               'Set-AzDeploymentManagerStep', 
+               'Remove-AzDeploymentManagerStep'
+
+# Variables to export from this module
+# VariablesToExport = @()
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = @()
+
+# DSC resources to export from this module
+# DscResourcesToExport = @()
+
+# List of all modules packaged with this module
+# ModuleList = @()
+
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','ResourceManager','ARM','DeploymentManager'
+
+        # A URL to the license for this module.
+        LicenseUri = 'https://aka.ms/azps-license'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/Azure/azure-powershell'
+
+        # A URL to an icon representing this module.
+        # IconUri = ''
+
+        # ReleaseNotes of this module
+        ReleaseNotes = '* First release for private preview'
+
+        # Prerelease string of this module
+        Prerelease = 'preview'
```

Comment:
@deveshguha you can comment this field out since a 0.x.x represents a preview module